cmd/edgectl/aes_install.go

```diff
@@ -453,6 +458,18 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
+	versions, err := i.SilentCaptureKubectl("get versions", "", "version", "-o", "json")
```

Comment:
I don't think this should be silent. The output is not that long and would be handy even for a user, e.g., to notice a surprising version number as a means of recognizing that they are using the wrong cluster.